AN00209_xCORE-200_DSP_Library/app_fft/src/app_fft.xc

```diff
@@ -22,186 +22,222 @@
 #define PRINT_CYCLE_COUNT 1
 #endif
 
-#define INPUT_FREQ N_FFT_POINTS/8
+#define INPUT_FREQ N_FFT_POINTS / 8
 
 #ifndef INT16_BUFFERS
 #define INT16_BUFFERS 0 // disabled by default. int32_t buffers is default
 #endif
 
 #ifndef TWOREALS
-#define TWOREALS 0 // Processing two real signals with a single complex FFT. Disabled by default
+#define TWOREALS \
+  0 // Processing two real signals with a single complex FFT. Disabled by
+    // default
 #endif
 
 #if TWOREALS
 int32_t do_tworeals_fft_and_ifft();
 #else
-int32_t do_complex_fft_and_ifft();
+int32_t       do_complex_fft_and_ifft();
 #endif
 
 // Array holding one complex signal or two real signals
 #if INT16_BUFFERS
 dsp_complex_short_t data[N_FFT_POINTS];
 #else
-dsp_complex_t  data[N_FFT_POINTS];
+dsp_complex_t data[N_FFT_POINTS];
 #endif
 
 
 /**
- * Experiments with functions that generate sine and cosine signals with a defined number of points
+ * Experiments with functions that generate sine and cosine signals with a
+ *defined number of points
```

Comment:
Is it possible to have it add whitespace after the `*` when it breaks multiline comments like this?

Reply:
This should be the default, not sure what happened here. It works correctly here (https://github.com/xmos/lib_dsp/pull/119/files#diff-9752432e546735d3fb65c7ec27346416R215) for example.